connection-limit/connection_limit_kern.c

```diff
@@ -273,18 +273,23 @@ int _xdp_limit_conn(struct xdp_md *ctx)
     if (data + sizeof(struct ethhdr)+ 1 > data_end)
         return XDP_PASS;
 
-    /* Check if its valid ip packet */
+    
     struct iphdr *iph = (struct iphdr *)(data + sizeof(struct ethhdr));
```

Comment:
Looks like a bug, is there someplace that handles IPv6?

Reply:
@dthaler You are right. There is no IPv6 handling as of now. Check for IPv6 packets is missing in connection-limiting, which is present in rate-limiting program. Below is the snippet from _xdp_ratelimit() where they are ignoring other than IPv4 packets.
```
    uint16_t eth_type = eth->h_proto;
    if (ntohs(eth_type) != ETH_P_IP) {
        return XDP_PASS;
    }
```

Reply:
Yes, that's what is needed here until this supports IPv6.  I've marked this PR as "changes requested" until that code is added here.